Modules/Segmentation/ConnectedComponents/test/itkRelabelComponentImageFilterSignednessTest.cxx

```diff
@@ -0,0 +1,33 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkImage.h"
+#include "itkRelabelComponentImageFilter.h"
+#include "itkTestingMacros.h"
+
+using PixelType = signed short;
+
+// Explicit template instantiation to test compile-time support of signed types
+template class itk::RelabelComponentImageFilter<itk::Image<PixelType>, itk::Image<PixelType>>;
```

Comment:
Thank you very much for following my suggestion, Jon 👍 Do you still think it's useful to keep it separate from the existing itkRelabelComponentImageFilterTest.cxx now? I think you might as well just move the template instantiation to either itkRelabelComponentImageFilterTest.cxx or itkRelabelComponentImageFilterGTest.cxx (the corresponding GoogleTest file)

Reply:
See 4ba0ab0. Thanks Niels.